vector/src/main/java/im/vector/app/features/settings/devices/v2/list/OtherSessionsView.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.settings.devices.v2.list
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.constraintlayout.widget.ConstraintLayout
+import dagger.hilt.android.AndroidEntryPoint
+import im.vector.app.R
+import im.vector.app.core.extensions.configureWith
+import im.vector.app.databinding.ViewOtherSessionsBinding
+import im.vector.app.features.settings.devices.DeviceFullInfo
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class OtherSessionsView @JvmOverloads constructor(
+        context: Context,
+        attrs: AttributeSet? = null,
+        defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    @Inject lateinit var otherSessionsController: OtherSessionsController
+
+    private val views: ViewOtherSessionsBinding
```

Comment:
We can use the simpler following syntax for view binding:

```
private val views = ViewOtherSessionsBinding.inflate(
    LayoutInflater.from(context),
    this
) 
```
You will need to use a `merge` tag as parent in `view_other_sessions` with `tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout"`.
This avoids uncesseary empty root view in the custom view (right now there is a `ConstraintLayout` inside a `ConstraintLayout`).
The same remark applies for the `CurrentSessionView`.

Reply:
It couldn't draw background if we use `merge`.